python/hail/dataset.py

```diff
@@ -750,6 +750,63 @@ def annotate_variants_expr(self, condition):
         pargs = ['annotatevariants', 'expr', '-c', condition]
         return self.hc.run_command(self, pargs)
 
+    def annotate_variants_keytable(self, keytable, condition, vdsKey=None):
```

Comment:
Prefer _ to camelCase in python.

This should be a list of strings defaulting to ['v'].  The types of the `vds_keys` in order should match the types of the key table's keys, in order.

I think this will be lost on most, but for honestly and clarity's sake, you should probably explain that if `vds_keys != ['v']`, this dataset will be evaluated twice.